_submissions/round-12/2/2015-03-12-kyler-brown.md

```diff
@@ -0,0 +1,51 @@
+---
+date: 2015-03-12
+round: Round 12
+title: Boolean logic in Python MCQs
+author: Kyler Brown
+permalink: /2015/03/kyler-brown-mcq/
+tags:
+  - MCQ
+  - Python
+  - Boolean
+---
+
+Topic
+-------
+
+These MCQs are for the Boolean logic section of 
+lesson [Making Choices](https://swcarpentry.github.io/python-novice-inflammation/05-cond.html).
+
+First Multiple Choice Question
+------------------------------
+
+    if (10 < 0) or (0 < -10):
+        print("A")
+    elif (10 > 0) and False:
+        print("B")
+    else:
+        print("C")
+
```

Comment:
I don't understand: am I supposed to guess which of 'A', 'B', and 'C' this prints?